### PR TITLE
[MOL-12691] Update padding for chips/pills

### PR DESCRIPTION
**Pull request by Hilman-Azfar**
Update the padding for chip as advised from ux
-  delete branch

### CODE REVIEW

File: src/components/fields/textarea/textarea.styles.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ export const Wrapper = styled.div<{ chipPosition?: "top" | "bottom" | undefined
 `;
 
 export const ChipContainer = styled.div`
-	margin: 0.5rem 0rem;
+	margin: 0.5rem 0rem 1rem 0rem;
```
Comment (weili-govtech) [inline src/components/fields/textarea/textarea.styles.tsx:18]:
```suggestion
	margin: 0.5rem 0 1rem;
```

**Comment by weili-govtech** [comment]
does this change affect the `chips` component as well?
https://designsystem.life.gov.sg/web-frontend-engine/index.html?path=/story/field-chips--default

the chips component can come with the textarea too: https://designsystem.life.gov.sg/web-frontend-engine/index.html?path=/story/field-chips--with-textarea

**Comment by Hilman-Azfar** [comment]
the chips component is unaffected

File: src/components/fields/chips/chips.styles.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import styled from "styled-components";
 
-export const ChipContainer = styled.div`
+export const ChipContainer = styled.div<{ showTextarea?: boolean | undefined }>`
```
Comment (weili-govtech) [inline src/components/fields/chips/chips.styles.tsx:3]: rename as `$showTextarea` to mark it as a transient prop

File: src/components/fields/textarea/textarea.styles.tsx [hunk 3]
```diff
@@ -14,8 +14,8 @@ export const Wrapper = styled.div<{ chipPosition?: "top" | "bottom" | undefined
 	flex-direction: ${({ chipPosition }) => (chipPosition !== "bottom" ? "column" : "column-reverse")};
 `;
 
-export const ChipContainer = styled.div`
-	margin: 0.5rem 0rem;
+export const ChipContainer = styled.div<{ chipPosition?: "top" | "bottom" | undefined }>`
```
Comment (weili-govtech) [inline src/components/fields/textarea/textarea.styles.tsx:17]: rename as `$chipPosition` to mark it as a transient prop

**Review by weili-govtech — CHANGES_REQUESTED** [review]

**Review by weili-govtech — APPROVED** [review]

**Pull request merged**